docs/extending/clusterexplorernodes.md

```diff
@@ -193,6 +193,34 @@ a `NodeSource`, then use its `at` method to specify where to attach to the tree.
 The `at` method may specify an existing grouping folder (specified by display name), or
 `undefined` to appear directly under the context tree node.
 
+**`API v1_1`**
```

Comment:
Switching to an "API v1_1" heading here feels disorienting to me.  I suggest we leave the original content and flow, except for updating the code sample to include the `apiName` as you have done, and then add a note that the `apiName` argument is not available in v1.

The principle I'm thinking of it that the docs should reflect how _someone writing an extension now_ would use the API, with backward compatibility notes for people maintaining extensions that they don't want to port to a new API version.

That would mean we should shift the 'how to get the API object' text to do `await k8s.extension.clusterExplorer.v1_1` instead of `.v1`.  What do you reckon?

Reply:
Yeah, sounds good. I also changed the sentence where we explain what the `apiName` arg is used for. It's clearer now.